al_aws.js

```diff
@@ -33,59 +30,6 @@ var selfUpdate = function (callback) {
     });
 };
 
-var getMetricStatistics = function (params, statistics, callback) {
```

Comment:
Please keep these functions for backward compatibility as `cwe-collector` still use them. Just make them call new functions from ` statistics_templates`